src/client.ts

```diff
@@ -88,11 +89,6 @@ function default_backoff(attempt: number): number {
   return 2 ** attempt * 100 + Math.random() * 100;
```

Comment:
I think this is redundant now? Looks like it's been moved to options.ts

Reply:
Yes. Removed.